browser/index.mjs

```diff
@@ -54,6 +47,34 @@ const tokenIssues = (token) => {
   )
 }
 
+const alwaysTrue = always(true)
+const or = (before, token) => (issue) => before(issue) || token(issue)
+const and = (before, token) => (issue) => before(issue) && token(issue)
+const tokenMatches = (token) => {
+  token = token.startsWith('/') ? `${token.slice(1)}/` : token
```

Comment:
is this for converting to a relative directory?

Reply:
The query language uses `/<some-dir` but we want to match on `<some-dir>/` in the issue articles text.

Example

![2023-12-19-190946_179x61_scrot](https://github.com/bas080/issue/assets/1840380/1798226e-85d8-4856-999f-42a7c8319041)

You can see that the text that represents this path does not start with a `/`, but if I want to match on it I do `/active`. So a quick and dirty trick is to change the search token `/active` to `active/`. It also happens to exclude filepaths (which used to also match).